client/cli/Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Parity Technologies <admin@parity.io>"]
 edition = "2021"
 
 [dependencies]
-clap = { version = "3.2.22", features = ["derive", "deprecated"] }
+clap = { version = "4.0.9", features = ["derive", "deprecated"] }
```

Comment:
Do we still need the `deprecated` feature ?

Reply:
No, removed :+1: